_includes/sidebar/latest-posts.njk

```diff
@@ -13,7 +13,7 @@
 <section>
   <div class="mt-20 max-w-6xl mx-auto pb-8 md:px-4 sm:px-4 px-4">
     <div class="grid-cols-12 grid gap-15">
-      <div class="col-span-12 md:col-span-6">
+      <div class="col-span-12 md:col-span-7">
```

Comment:
I put the grid 7/5. 8/4 made it compact. But I decreased the width of "read more on" - looks better. But we can discuss it tomorrow.

Reply:
Ok